x-pack/plugins/ml/public/application/jobs/jobs_list/components/datafeed_chart_flyout/datafeed_chart_flyout.tsx

```diff
@@ -523,7 +523,7 @@ export const DatafeedChartFlyout: FC<DatafeedChartFlyoutProps> = ({
                           key={'job-results'}
                           color={euiTheme.euiColorAccentText}
                           id={i18n.translate('xpack.ml.jobsList.datafeedChart.bucketSeriesId', {
-                            defaultMessage: 'Job results',
+                            defaultMessage: 'Count of documents read by datafeed',
```

Comment:
Looking at what we use in the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/8.4/ml-get-bucket.html#ml-get-bucket-results) for `event_count`, I tried replacing `read` by `processed` but don't like the way it wraps on to 2 lines!

<img width="280" alt="image" src="https://user-images.githubusercontent.com/7405507/195895351-69c8e71e-b66c-4c1d-8189-33ba4770049d.png">

I can't think of a better, short alternative to what you have here. Over to you @lcawl !

Is there anything you can do to stop the truncation of these labels in the legend?

<img width="514" alt="image" src="https://user-images.githubusercontent.com/7405507/195895805-26033f46-e3a8-4165-ae7e-25da4cfe3b65.png">

Reply:
The truncation is built into the elastic charts `LineSeries` component - so a change would require us updating the style of the internal component directly 🤔 Not sure we want to do that here. Maybe we should change the tooltips so that we can still see 'Source indices' and 'Datafeed' at the start so that it's still clear even with truncation?
cc @lcawl

Reply:
Yup, putting the important phrases at the front LGTM